### DISCUSSION TITLE
How to search projects in github

### DISCUSSION THREAD

**Discussion by UG-SEP** [EddieHubCommunity/support#1898] [Tips and Tricks]
I have posted a new video that helps you to know that how to search projects in Github 
Hope you like it...
https://youtu.be/2B1VUo22hGA

**Comment by ghost** [comment]
We understand your enthusiasm @UG-SEP but @eddiejaoude and @kaiwalyakoparkar with other folks are trying the reduce the cluttering of these discussions, you can create a single discussion, and post all your yt videos/tweets there? Hope you understand!!

**Reply by UG-SEP** [reply]
ok

**Comment by UG-SEP** [comment]
Hey everyone, As I difficult for the beginner to understand Github I have created a video about Top Git+Github tricks and tips watch the video hope It helps to you all https://youtu.be/uISaVQafY88

**Comment by UG-SEP** [comment]
It is the most difficult for beginners to create Multiple Pull Request in GitHub they say that the Previous PR commit merges with it.                                                                                 
 So I have created a video that how to create Multiple PR to Github
I hope this help you
[Watch Video](https://youtu.be/aDyIjBb8z0w)
